htp/htp_decompressors.h

```diff
@@ -55,9 +55,7 @@ typedef struct htp_decompressor_t htp_decompressor_t;
 #define DEFLATE_MAGIC_2         0x8b
 
 struct htp_decompressor_t {
-    htp_status_t (*decompress)(htp_decompressor_t *, htp_tx_data_t *);
```

Comment:
this is a public or private struct? If it is public we can't remove from it.

Reply:
Indeed, restoring them